cli/src/pcluster/constants.py

```diff
@@ -66,13 +66,16 @@
 LUSTRE = "LUSTRE"
 OPENZFS = "OPENZFS"
 ONTAP = "ONTAP"
+FILECACHE = "FILECACHE"
 
 FSX_LUSTRE = "FsxLustre"
 FSX_OPENZFS = "FsxOpenZfs"
 FSX_ONTAP = "FsxOntap"
+FSX_FILE_CACHE = "FsxFileCache"
 
 # https://docs.aws.amazon.com/fsx/latest/APIReference/API_DescribeVolumes.html#FSx-DescribeVolumes-request-VolumeIds.
 FSX_VOLUME_ID_REGEX = r"^fsvol-[0-9a-f]{17}$"
+FSX_FILE_CACHE_ID_REGEX = r"^fc-[0-9a-f]{17}$"
```

Comment:
Minor: if possible, let's add a comment with the link to the FSx public docs where the regexp is defined.